client/src/components/ActiveFriends/ActiveFriends.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { Avatar, IconButton, Paper, Tooltip, Typography } from '@material-ui/core'
+import PropTypes from 'prop-types'
+import useStyles from './activeFriendsStyles'
+import Preloader from '../Preloader/Preloader'
+import { getAvatarLink } from '../../utils/helpers/imageLinkHelpers'
+import { getActiveTime } from '../../utils/date/getDate'
+import MailOutlineIcon from '@material-ui/icons/MailOutline'
+import { Link } from 'react-router-dom'
+
+const ActiveFriends = ({ activeFriends, activeFriendsAreLoading }) => {
+  const classes = useStyles()
+
+  const friendsList = () => {
+    if (activeFriends.length === 0) {
+      return <p className={classes.notification}>You have no active friends.</p>
+    } else {
+      return activeFriends.map(friend => (
+        <div className={classes.container} key={friend.lastActivityTime}>
```

Comment:
why the key is last activity time? It may change with time. Why not friend.username?